lcm/service/lcm/constants.go

```diff
@@ -77,9 +79,9 @@ const (
 	loadModelMilliCPU=20
 	loadModelMemInMB=50
 	loadTrainingDataMilliCPU=20
-	loadTrainingDataMemInMB=300
-	logCollectorMilliCPU=60
-	logCollectorMemInMB=300
+	loadTrainingDataMemInMB=100
+	logCollectorMilliCPU=20
+	logCollectorMemInMB=100
```

Comment:
Log collector may have memory insufficiency with only 100MB memory. We may want to increase the constant for now or make it configurable using viper.